samplers/samplers.go

```diff
@@ -166,14 +168,15 @@ type JSONMetric struct {
 
 // Counter is an accumulator
 type Counter struct {
-	Name  string
-	Tags  []string
-	value int64
+	Name   string
+	Tags   []string
+	value  float64
+	counts int64
```

Comment:
Can we add a comment explaining `counts` vs. `value`? (now that `Counter.counts` is not the same as `Counter.value`).